packages/forms-web-app/src/app.js

```diff
@@ -87,13 +89,26 @@ if (config.server.useSecureSessionCookie) {
   app.set('trust proxy', 1); // trust first proxy
 }
 
+let sessionStoreConfig = sessionConfig();
+
+if (config.featureFlag.useSessionStore) {
+  const redisClient = createClient({ url: 'redis://redis:6379' });
```

Comment:
Can we make the redis URL a config param too please?

Reply:
done